### BIG differences, this is diff of wrong files.
### CLUS "System Class CHARACTER"
### and CLHS topic selection http://clhs.lisp.se/Body/a_ch.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:character.txt	2017-04-27 22:03:47.000000000 +0100
+++ saved-pages-clus/cl:types:character.txt	2017-04-27 22:03:47.000000000 +0100
@@ -1,4 +1,25 @@
-CHARACTER
-Please select which reference to CHARACTER you intended
+System Class CHARACTER

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Class Precedence List
+character, t
+Description
+A character is an object that represents a unitary token in an aggregate quantity of text; see section {\secref\CharacterConcepts}.
+The types base-char and extended-char form an exhaustive partition of the type character.
+See Also
+{\secref\CharacterConcepts}, {\secref\SharpsignBackslash}, {\secref\PrintingCharacters}
+\
\ No newline at end of file
